stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewClient.kt

```diff
@@ -42,7 +42,7 @@ internal class PaymentAuthWebViewClient(
             onAuthCompleted(
                 runCatching {
                     Uri.parse(url)
-                }.getOrNull()
```

Comment:
are we intentionally swallow the exception here?

Reply:
Good catch, my change is incorrect.